src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -119,21 +128,23 @@
           constants/content-type-album   [album/album-message message-data context]
 
           [not-implemented/not-implemented [content.unknown/unknown-content message-data]])
-        [status/status message-data]]]]]))
+        (when @show-delivery-state?
+          [status/status outgoing-status])]]]]))
 
 (defn message-with-reactions
   [{:keys [pinned pinned-by mentioned in-pinned-view? content-type
            last-in-group? message-id]
     :as   message-data}
    {:keys [chat-id] :as context}]
-  [rn/view
-   {:style               (style/message-container in-pinned-view? pinned mentioned last-in-group?)
-    :accessibility-label :chat-item}
-   (when pinned
-     [pin/pinned-by-view pinned-by])
-   (if (#{constants/content-type-system-text constants/content-type-community
-          constants/content-type-contact-request}
-        content-type)
-     [system-message-content message-data]
-     [user-message-content message-data context])
-   [reactions/message-reactions-row chat-id message-id]])
+  (let [show-delivery-state? (reagent/atom false)]
```

Comment:
It always confuses me to see a reactive atom without a proper form-2 component. I've only seen it in the Status codebase. In React this kind of stuff is acceptable because `useState` is a hook with completely different capabilities, but in Reagent it's a different story.

There's probably a better way than resetting the state on every re-render of `message-with-reactions`. Even if it works @briansztamfater, this kind of practice can easily lead to bad performance due to the recreation of atoms on every render pass, and thus re-rendering of the entire sub-tree.

It's a dangerous practice. I also couldn't find a single example in Reagent docs and source examples doing this. Something tells me this is wrong.